src/me/ryanhamshire/PopulationDensity/RegionCoordinates.java

```diff
@@ -53,42 +53,39 @@ public static RegionCoordinates fromLocation(Location location)
 
     //converts a string representing region coordinates to a proper region coordinates object
     //used in reading data from files and converting filenames themselves in some cases
-    public RegionCoordinates(String string)
-    {
+    public RegionCoordinates(String string) {
         //split the input string on the space
         String[] elements = string.split(" ");
 
         //expect two elements - X and Z, respectively
-        String xString = elements[0];
-        String zString = elements[1];
-
         //convert those to integer values
-        this.x = Integer.parseInt(xString);
-        this.z = Integer.parseInt(zString);
+        this.x = Integer.parseInt(elements[0]);
+        this.z = Integer.parseInt(elements[1]);
     }
 
     //opposite of above - converts region coordinates to a handy string
-    public String toString()
-    {
-        return Integer.toString(this.x) + " " + Integer.toString(this.z);
+    public String toString() {
+        return new StringBuilder()
+            .append(this.x)
+            .append(" ")
+            .append(this.z)
+            .toString();
     }
 
     //compares two region coordinates to see if they match
     @Override
-    public boolean equals(Object coordinatesToCompare)
-    {
+    public boolean equals(Object coordinatesToCompare) {
         if (coordinatesToCompare == null) return false;
 
         if (!(coordinatesToCompare instanceof RegionCoordinates)) return false;
 
-        RegionCoordinates coords = (RegionCoordinates)coordinatesToCompare;
+        RegionCoordinates coords = (RegionCoordinates) coordinatesToCompare;
```

Comment:
minor style thing, but seems inconsistent, as far as I know, for explicit casts there is no space.

Reply:
okay, I'm using the default styler from IntelliJ and used to whitespaces after ")" and add it unintentionally when i see it. But I will reset the styling to keep it consistent.